GliaWidgets/Component/Header/Header.swift

```diff
@@ -50,23 +57,31 @@ class Header: UIView {
         updateHeight()
     }
 
-    func setLeftItem(_ item: UIView?, animated: Bool) {
-        setItem(item, to: leftItemContainer, animated: animated)
+    public func setLeftItem(animated: Bool) {
+        setItem(backButton, to: leftItemContainer, animated: animated)
     }
```

Comment:
1. No need to make it public. `Header` itself is an `internal` class, so everything in it can only be `internal` or stricter (`fileprivate` and `private`). Even though it compiles without warnings, it does not make sense to declare methods or fields as `public`.
2. If this method always sets only the back button, the name of the function should reflect that - it's not `setLeftItem` anymore, rather `addBackButton`.

Reply:
Actually, looking through the other bits of code, I'm not even sure this function should behave as it does. 

If the left container only ever contains a back button, why not add it straight away, just as you do with the `endButton`, `endScreenShareButton` and `closeButton` in `layout()`? And then create a function `showBackButton` which only affects the `isHidden` attribute and call it?